app/javascript/components/align-and-fine-tune/stores/mappingStore.js

```diff
@@ -69,14 +69,16 @@ export const noMatchPredicate = (predicate) =>
   (isString(predicate) && predicate.toLowerCase().includes('no match')) ||
   predicate?.source_uri?.toLowerCase()?.includes('nomatch');
 
-const filterTerms = (terms, inputValue, options = { pickSelected: false }) =>
-  terms
-    .filter(
-      (term) =>
-        (options.pickSelected ? term.selected : !term.selected) &&
-        term.name.toLowerCase().includes(inputValue.toLowerCase())
-    )
+const filterTerms = (terms, inputValue, options = { pickSelected: false }) => {
+  const searchValue = inputValue.toLowerCase();
+  const searchBy = (term) =>
+    term.name.toLowerCase().includes(searchValue) ||
```

Comment:
Looks like both term names and property comments can contain `NULL`s. Maybe those two should be coalesced to `""` unless it already happens somewhere in the serializer?

Reply:
nice catch, will change to lodash `toLower`, name can't be blank, but other fields can. 
as about serializers we're using in-built one, thinking about adding `active_model_serializers` or `jsonapi-serializer`, any thoughts on this?

Reply:
@excelsior fix was force pushed